models/tests/test_plan_statistics.py

```diff
@@ -0,0 +1,350 @@
+        test_camp.volunteers.add(volunteer_a, volunteer_b, volunteer_c)
+
+        volunteer_count = ps.count_volunteers(test_camp)
+
+        self.assertEqual(volunteer_count, 3)
+
+    def test_deactivated_volunteer_not_counted(self):
+        """
+        Test case where only deactive volunteers are in the file.
+        If this passes, volunteer_count should equal 0, since deactivated volunteers are not counted in this function.
+        """
+        Plan(name='test_plan4',
+             emergency_type=Plan.EmergencyType.EARTHQUAKE,
+             description='Test emergency plan',
+             geographical_area='London',
+             camps=[Camp(name='camp4')])
+        volunteer_a = Volunteer(username='yunsy', password='root', firstname='Yunsy', lastname='Yin',
+                                phone='+447519953189')
+        volunteer_b = Volunteer(username='paul', password='root', firstname='Paul', lastname='Shoemaker',
+                                phone='+447519955439')
+        volunteer_c = Volunteer(username='gerald', password='root', firstname='Gerald', lastname='Smith',
+                                phone='+447511111111')
+        test_plan = Plan.find('test_plan4')
+        test_camp = test_plan.camps.get('camp4')
+        test_camp.volunteers.add(volunteer_a, volunteer_b, volunteer_c)
+        volunteer_a.account_activated = False
+        volunteer_b.account_activated = False
+        volunteer_c.account_activated = False
+
+        volunteer_count = ps.count_volunteers(test_camp)
+        self.assertEqual(volunteer_count, 0)
+
+    def test_available_volunteer_count(self):
+        """
+        Test case where only active volunteers are in the file.
+        If this passes, volunteer_count and number of volunteers added in test case should be equal.
+        Expecting volunteer_count = 3.
+        """
+        Plan(name='test_plan4',
+             emergency_type=Plan.EmergencyType.EARTHQUAKE,
+             description='Test emergency plan',
+             geographical_area='London',
+             camps=[Camp(name='camp5')])
+        volunteer_a = Volunteer(username='yunsy', password='root', firstname='Yunsy', lastname='Yin',
+                                phone='+447519953189')
+        volunteer_b = Volunteer(username='paul', password='root', firstname='Paul', lastname='Shoemaker',
+                                phone='+447519955439')
+        volunteer_c = Volunteer(username='gerald', password='root', firstname='Gerald', lastname='Smith',
+                                phone='+447511111111')
+        test_plan = Plan.find('test_plan4')
+        test_camp = test_plan.camps.get('camp5')
+        test_camp.volunteers.add(volunteer_a, volunteer_b, volunteer_c)
+
+        volunteer_count = ps.count_volunteers(test_camp)
+
+        self.assertEqual(volunteer_count, 3)
+
+    def test_unavailable_volunteer_not_counted(self):
+        """
+        Test case where only deactive volunteers are in the file.
```

Comment:
I believe this should say: 
Test case where only unavailable volunteers are in the file.
If this passes, volunteer_count should be equal to 0, since unavailable volunteers are not counted in this function. 

Since this is a test regarding availability.